docs/contributors/release.md

```diff
@@ -114,13 +114,16 @@ Creating a release involves:
 
 1. Checkout the release branch `git checkout release/x.x`.
 
-**Note:** This branch should never be removed or rebased. This means in case of conflicts when creating PRs from this branch, create temporary branches in order to merge these PRs and avoid touching the release branch.
+**Note:** This branch should never be removed or rebased. When we want to merge something from it to master and conflicts exist/may exist we use a temporary branch `bump/x.x`.
 
 2. Create [a commit like this](https://github.com/WordPress/gutenberg/commit/00d01049685f11f9bb721ad3437cb928814ab2a2#diff-b9cfc7f2cdf78a7f4b91a753d10865a2), removing the `-rc.X` from the version number in `gutenberg.php`, `package.json`, and `package-lock.json`.
-3. Create a Pull Request from the release branch into `master` using the changelog as a description and ensure the tests pass properly.
-4. Tag the version. `git tag vx.x.0` from the release branch.
-5. Push the tag `git push --tags`.
-6. Merge the version bump pull request and avoid removing the release branch.
+3. Tag the version. `git tag vx.x.0` from the release branch.
```

Comment:
I think we should add an entry before this to ensure the tests are ok. The PR was used for it but maybe we could do it manually instead? or create a temporary PR?

Reply:
And should we do the same updates done here for the RC release?

Reply:
The process was updated to make sure the CI tests pass before creating the tag.

> And should we do the same updates done here for the RC release?
For the RC release, I don't think we need a temporary branch as in that case there are no conflicts.